docs/src/components/footer/EmailSubscribe.tsx

```diff
@@ -109,23 +108,44 @@ export default function EmailSubscribe({ sx }: { sx?: SxProps<Theme> }) {
           onChange={(event) => setForm({ email: event.target.value, status: 'initial' })}
           inputProps={{ required: true }}
           sx={{
+            minWidth: 220,
             bgcolor: (theme) =>
+              theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : '#fff',
+            boxShadow: (theme) =>
               theme.palette.mode === 'dark'
-                ? theme.palette.primaryDark[900]
-                : theme.palette.grey[100],
+                ? '0 1px 2px 0 rgba(0 0 0 / 1)'
+                : '0 1px 2px 0 rgba(0 0 0 / 0.1)',
+            borderRadius: 1,
+            border: '1px solid',
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark'
+                ? theme.palette.primaryDark[500]
+                : theme.palette.grey[300],
             px: 1,
             py: 0.5,
```

Comment:
A fun note on adding padding in the div and not the input, here is how it looks on forms, with the autocompelte:

<img width="364" alt="Screenshot 2022-07-28 at 02 08 54" src="https://user-images.githubusercontent.com/3165635/181393162-686a35ff-640d-4f47-86cb-42d2965f3c3f.png">

I think that it doesn't care for the homepage, it's great as is, but it can be interesting to know, e.g. for use in Joy UI:

<img width="237" alt="Screenshot 2022-07-28 at 02 11 52" src="https://user-images.githubusercontent.com/3165635/181393438-7d2ebaac-af01-44d8-8289-f5a647e63f6a.png">

Reply:
Another opportunity about this one, dark mode: #17863.

<img width="379" alt="Screenshot 2022-08-21 at 01 29 41" src="https://user-images.githubusercontent.com/3165635/185769338-0f6ce898-fb56-4289-9803-9915b0180f13.png">

raised by @prakhargupta1 in https://github.com/mui/mui-toolpad/pull/809#issuecomment-1220307463.